openvpn/config.sls

```diff
@@ -8,6 +8,14 @@ include:
 {% if type in ['client', 'server', 'peer'] %}
 {% for name, config in names.iteritems() %}
 
+{% macro _permissions(mode=644, user=None, group=None) %}
+{%-  if not grains['os_family'] == 'Windows' -%}
+    - mode: {{ mode }}
+    - user: {% if user %}{{ user }}{% elif config.user is defined %}{{ config.user }}{% else %}{{ map.user }}{% endif %}
+    - group: {% if group %}{{ group }}{% elif config.group is defined %}{{ config.group }}{% else %}{{ map.group }}{% endif %}
```

Comment:
`if group`: When `group` is `0` this evaluates to false. I'll fix that.

Reply:
Good catch! Thanks